include/fmt/format-inl.h

```diff
@@ -968,16 +969,27 @@ FMT_FUNC void report_windows_error(int error_code,
 #endif
 
 FMT_FUNC void vprint(std::FILE* f, string_view format_str, format_args args) {
+  if (std::fwide(f, -1) >= 0) {
```

Comment:
What happens if we omit this. The data will be interpreted as a sequence of `wchar_t`s?

Could you add a unit test covering the case of mixing wide and multibyte writes to the same stream?

Reply:
If `fwrite` is called on a wide-oriented stream, it will not write any bytes under glibc (and likely other POSIX C implementations). This will result in fwrite_fully's length exception being thrown.

Reply:
What condition should a unit test look for? Would it verify that an exception is thrown on non-Windows platforms? (windows allows mixing and ultimately has no notion of stream orientation evidenced by MSDN's `fwide` documentation).

Reply:
Let's just drop this per comments below.